app/models/list.rb

```diff
@@ -0,0 +1,10 @@
+class List < ActiveRecord::Base
+  has_many :list_entries
```

Comment:
I'm not sure. If this was a 'real' app, I would hold off adding that until working on the story for deleting lists, but for this sample project I don't think we have any such story. What do you think @pedrosmmoreira ?